packages/tasks/src/model-libraries.ts

```diff
@@ -725,6 +725,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.transformersJS,
 		filter: true,
 	},
+	"trellis": {
+		prettyLabel: "Trellis",
+		repoName: "Trellis",
+		repoUrl: "https://github.com/microsoft/TRELLIS",
+		countDownloads: `path:"*/model.safetensors"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"safetensors"`,
```

this seems more appropriate to based on https://huggingface.co/JeffreyXiang/TRELLIS-image-large/tree/main/ckpts, isn't it?

Reply:
Would this count any download at https://huggingface.co/JeffreyXiang/TRELLIS-image-large/tree/main/ckpts?